addons/point_of_sale/static/src/js/models.js

```diff
@@ -1810,7 +1801,7 @@ class Orderline extends PosModel {
         return round_pr(this.get_unit_price() * this.get_quantity() * (1 - this.get_discount()/100), rounding);
     }
     get_taxes_after_fp(taxes_ids){
-        return this.pos.get_taxes_after_fp(taxes_ids);
+        return this.pos.get_taxes_after_fp(taxes_ids, this.order.fiscal_position);
```

Comment:
as we are in a refactoring in master, you can delete this function on posorderline and use the one on POS where this one was called/

Reply:
Done. I added a temporary commit to highlight the change for this comment.